cwd/app.js

```diff
@@ -269,16 +271,18 @@
        * This makes it so that the factory cannot work to add data to gauges.
        * Instead, we need to add the data while handling the view object.
        */
+      let resStatus = 0;
       if (obj.view && obj.view.gauges) {
         obj.view.gauges.forEach((g, index) => {
-          if (g.data_url) {
+           if (g.data_url) {
             fetch(g.data_url)
-            .then(r => r.json())
+            .then(r => {
+              r.json();
+            })
             .then(j => {
               g.data = j;
-
               // Then make call to store the new data into the database.
-              fetch(`http://${API_URL}/glyphs/${obj._id}/gauges/${index + 1}/cache`, {
+              /*fetch(`http://${API_URL}/glyphs/${obj._id}/gauges/${index + 1}/cache`, {
```

Comment:
Why did you comment out the cache?

Reply:
Outdated, should be fixed in the most recent commit

Reply:
Never mind